Formula/sbt.rb

```diff
@@ -20,11 +20,24 @@ def install
       s.gsub! "/etc/sbt/sbtopts", "#{etc}/sbtopts"
     end
 
-    inreplace "bin/sbt-launch-lib.bash", "${sbt_home}/bin/sbt-launch.jar", "#{libexec}/sbt-launch.jar"
+    inreplace "bin/sbt-launch-lib.bash" do |s|
+      s.gsub! "$(dirname \"$(realpath \"$0\")\")", libexec
+      s.gsub! "$(dirname \"$sbt_bin_dir\")", libexec
+      s.gsub! "${sbt_home}/bin/sbt-launch.jar", "#{libexec}/sbt-launch.jar"
+      ## This is required to pass the test
```

Comment:
`$ brew test sbt` that calls `("#{bin}/sbt sbt-version")` fails to detect the java version correctly. It prints out

```
Picked up _JAVA_OPTIONS: -Dsbt.log.noformat=true -Duser.home=/Users/eugene/Library/Caches/Homebrew/java_cache
```